requirements.txt

```diff
@@ -6,7 +6,7 @@ scipy>=1.4.0
 torch>=1.7
 torchaudio
 soundfile
-librosa==0.8.0
+librosa>=0.10.0
```

Comment:
we need to bind it to ==0.10.0 to prevent incompat issues in the future.

Reply:
Yep, I agree. Would you be happy with `librosa==0.10.0.*` as there is technically a post1 and post2 version available?